Description: Fix Pathbar naming of Home
 The Pathbar replaces $HOME with "Home", i.e. Home/Desktop.
 This is frequently displayed as /home/Home/Desktop. This patch
 fixes said issue.
 .
 nautilus (1:3.4.2-1.4) two; urgency=low
 .
   * Fixed pathbar replacement of Home variable, i.e. /home/username/ as opposed to /home/Home/
Author: Ikey Doherty <ikey@solusos.com>

---
Reviewed-By: Ikey Doherty <ikey@solusos.com>
Last-Update: <2012-07-07>

--- nautilus-3.4.2.orig/src/nautilus-pathbar.c
+++ nautilus-3.4.2/src/nautilus-pathbar.c
@@ -1269,8 +1269,13 @@ get_dir_name (ButtonData *button_data)
 {
 	if (button_data->type == DESKTOP_BUTTON) {
 		return _("Desktop");
+	/*
+	}
+	 * originally this would look like /home/Home/Desktop in the pathbar.
+	 * I can see the logic, when you're only staying in $HOME, i.e. Home/Desktop
+	 * but when you've come from a directory further up it just looks wrong. *
 	} else if (button_data->type == HOME_BUTTON) {
-		return _("Home");
+		return _("Home");*/
 	} else {
 		return button_data->dir_name;
 	}
